### ISSUE TITLE
"stop all simulations" button doesn't work

### ISSUE THREAD

**Issue by jasmainak** [jonescompneurolab/hnn#284]
I'm on the master branch and running a simulation. When I click "Stop all simulations", it doesn't do anything.

I have this problem on Mac.

**Comment by blakecaldwell** [comment]
Are you using hnn-core master?

**Comment by blakecaldwell** [comment]
Also this may be needed:
https://github.com/jonescompneurolab/hnn-core/pull/297

**Comment by blakecaldwell** [comment]
There are several moving parts right now. Let's test this after a few more things have been merged. I also realized this is relevant:
https://github.com/jonescompneurolab/hnn/pull/282/commits/afae29dcdcd97082325b594da8cc84584d5b197c

**Comment by blakecaldwell** [comment]
Master of both hnn and hnn-core should have the fixes for this now!

**Comment by jasmainak** [comment]
Nopes, doesn't work for me yet. Btw, I'm using `JoblibBackend`. Could that be the issue?

**Comment by blakecaldwell** [comment]
> Nopes, doesn't work for me yet. Btw, I'm using `JoblibBackend`. Could that be the issue?

Yeah, that would be why. Haha, I hadn't even considered that. Instructions for HNN GUI are to use the MPIBackend of hnn-core. Can you test with MPI?
